lib/process-event.js

```diff
@@ -129,8 +129,25 @@ function processEvent(framework, body, name = '') {
       return framework.webex.messages.get(data.id)
         .then(message => {
           framework.myEmit('messageCreated', message);
+          // check if message is from bot...
```

Comment:
Here is the moved code...

There IS a side effect here—the command `bot.lastActivity = moment().utc().format();`, which happens just before the deleted logic in onMessageCreated, is no longer called when the framework processes a message sent by the bot.

Upon reflection, this is the correct behavior.